src/main/java/com/facebook/presto/maven/ServiceDescriptorGenerator.java

```diff
@@ -76,7 +76,7 @@ public void execute()
             pluginClasses = findImplementationsOf(Plugin.class, loader);
         }
         catch (Exception e) {
-            throw new MojoExecutionException(format("%n%nError scanning for classes implementing %s.", Plugin.class.getName()));
+            throw new MojoExecutionException(format("%n%nError scanning for classes implementing %s, %s", Plugin.class.getName(), e));
```

Comment:
why not emboed `e` as `cause` in `MojoExecutionException`? Isn't it printed then?